lib/utils/ConfigParser.js

```diff
@@ -14,7 +14,7 @@ const DEFAULT_CONFIGS = {
     baseUrl: null,
     waitforTimeout: 1000,
     framework: 'mocha',
-    reporter: 'dot',
+    reporter: [],
```

Comment:
if we do this change, we need to remove the dot reporter from the dependencies. I thought it would be nice to have it as default reporter so people don't get confused when they don't see anything in the terminal